### PR TITLE
add matrix for class YYPowGate

### PR DESCRIPTION
**Pull request by ishanrai05**
ref: #2806

### CODE REVIEW

**Comment by ishanrai05** [comment]
@vtomole

**Comment by ishanrai05** [comment]
@vtomole  Done

File: cirq/ops/parity_gates.py [hunk 1]
```diff
@@ -39,6 +39,16 @@ class XXPowGate(eigen_gate.EigenGate,
               [0 1 0 0]
               [1 0 0 0]
 
+    The XX**t gate implements the following unitary:
```
Comment (vtomole) [inline cirq/ops/parity_gates.py]: To be consistent with existing documentation, let's do it in this manner: https://github.com/quantumlib/Cirq/blob/144e1ebdd6196cf6c8ea8e6f497dbb947bf57142/cirq/ops/common_gates.py#L56

File: cirq/ops/parity_gates.py [hunk 2]
```diff
@@ -124,7 +134,24 @@ def __repr__(self) -> str:
...
+        At exponent=1, this gate implements the following unitary:
+
+        Y⊗Y = [ 0  0  0 -1]
+              [ 0  0  1  0]
+              [ 0  1  0  0]
+              [-1  0  0  0]
+
+        The YY**t gate implements the following unitary:
+
+        (Y⊗Y)^t = [cos(y) . . isin(y)]
```
Comment (vtomole) [inline cirq/ops/parity_gates.py]: Let's do something similar to https://github.com/quantumlib/Cirq/blob/144e1ebdd6196cf6c8ea8e6f497dbb947bf57142/cirq/ops/common_gates.py#L230

**Comment by vtomole** [comment]
Ping @ishanrai05

**Comment by vtomole** [comment]
@ishanrai05 Ping

File: cirq/ops/parity_gates.py (hunk 1 above)
Comment (dabacon) [reply cirq/ops/parity_gates.py:42]: I don't think this got resolved.  Was your latest commit uploaded?

**Comment by balopat** [comment]
Friendly ping @ishanrai05 - are you planning to finish this? I'd rather close it and you can reopen it when you have time to work on it. Thank you!

**Comment by balopat** [comment]
Closing due to inactivity, please reopen later - we might add this in a separate PR

**Pull request closed without merging**